modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+= Enabling the Dex openshift OAuth Connector
+
+Dex uses the users and groups defined within OpenShift by querying the platform provided `OAuth` server.
+
+. The `openShiftOAuth` property is used to trigger the Operator to auto configure the built-in OpenShift `OAuth` server.
+. The `groups` property is used to mandate the users to be part of one or all the groups in the groups list.
```

Comment:
Use the `openShiftOAuth` property to trigger the Operator to auto configure the built-in OpenShift `OAuth` server. 

Also, should it be `OpenShiftOAuth`? Please confirm as it surprises me that the S is capitalized but not the O.